src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -382,6 +382,10 @@ private void copySubstrateLibraries() throws IOException {
             Path freetypeLibPath = projectLibsLocation.resolve("libfreetype.so");
             Files.copy(javafxFreetypeLibPath, freetypeLibPath, StandardCopyOption.REPLACE_EXISTING);
         }
+        String srcd = this.ndk+"/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/lib/aarch64-linux-android/libc++_shared.so";
```

Comment:
Probably better to refactor in the constructor:
```
this.hostPlatformFolder = Paths.get(this.ndk, "toolchains", "llvm", "prebuilt", configuration.getHostTriplet().getOsArch()).toString();
...
```
so you can do:
```
Path srcd = Paths.get(hostPlatformFolder, "sysroot", "usr", "lib", ANDROID_TRIPLET, "libc++_shared.so");
```
Also you don't check if the file exists?